frontend/src/app/home/home.component.html

```diff
@@ -1,39 +1,81 @@
-<app-hero-slides [slides]=heroSlides></app-hero-slides>
-<div class="container mat-typography">
+<div
+  id="hero"
+  class="gradient-overlay container with-escaping-tiles"
+>
+  <div class="hero-content">
+    <h1>Transform Autism Outcomes. Together.</h1>
+    <hr/>
+    <p class="h4">
+      We're bringing together researchers, professionals, and community organizations to improve the lives of individuals
+      with autism and their families. Autism DRIVE gives you access to ground-breaking autism research and innovative
+      models for care, support, and education.
+    </p>
+  </div>
+
   <div
-    class="row action"
-    fxLayout="column"
-    fxLayoutAlign="center center"
+    class="row escaping-tiles"
+    fxLayout.lt-sm="column"
+    fxLayout.gt-xs="row wrap"
+    fxLayoutGap="40px"
+    fxLayoutAlign="center auto"
   >
-    <p class="intro">Improving the lives of individuals with autism and their families through ground-breaking research
-      and innovative models for care, support, and education.</p>
-    <div id="cta">
-      <button
-        mat-flat-button
-        (click)="goEnroll($event)"
-        color="primary"
-        class="btn-lg"
+    <div
+      class="border-box-tile"
+      fxFlex.gt-sm="calc(33% - 160px)"
+      fxFlex.sm="calc(50% - 120px)"
+    >
+      <div
+        class="border-box-tile-inner"
       >
-        Join the STAR Community
-      </button>
-      <button
-        mat-flat-button
-        (click)="goResources($event)"
-        color="accent"
-        class="btn-lg"
+        <div class="border-box-tile-label">Join Autism Drive</div>
```

Comment:
These boxes need to link to the appropriate pages

Reply:
These links seem to work now.